mono/mini/mini-exceptions.c

```diff
@@ -1446,6 +1454,21 @@ mono_crash_reporting_register_native_library (const char *module_path, const cha
 	native_library_whitelist = g_list_append (native_library_whitelist, entry);
 }
 
+static void
+mono_crash_reporting_batch_register_native_libraries (const char *module_string)
+{
+	// Accepts a ":"-delimited string where each pair of entries is module_path + module_string
+	// Example: "libsystem_pthread.dylib:pthread:libsystem_platform.dylib:platform" will register two modules ("pthread" and "platform")
+	char **tokens = g_strsplit (module_string, G_SEARCHPATH_SEPARATOR_S, 0);
+	while (*tokens) {
+		char *module_path = *tokens;
+		tokens++;
```

Comment:
```suggestion
		tokens++;
		g_assert (*tokens);
```
My comment got lost.  I think we could use an assert here in case the input is malformed.